scripts/run_lighthouse_tests.py

```diff
@@ -150,10 +174,7 @@ def main(args=None):
 
     if lighthouse_mode == LIGHTHOUSE_MODE_PERFORMANCE:
         python_utils.PRINT('Building files in production mode.')
-        # We are using --source_maps here, so that we have at least one CI check
```

Comment:
Note: Performance tests seem to be failing due to OOM issues, hence dropping --source_maps.